src/react-app-env.d.ts

```diff
@@ -0,0 +1,7 @@
+/// <reference types="react-scripts" />
+
+// TODO: Contribute type definitions for 'global' package to
```

Comment:
have you filed an issue for this? i have a general rule of thumb to link issues to TODO notes to attempt to prevent them from being lost

Reply:
Good idea! Done in https://github.com/jeremyckahn/farmhand/commit/24f0a6ae84d5261217b8e24dae63995b92ea1d4e.